src/Idris/AbsSyntax.hs

```diff
@@ -434,7 +434,7 @@ pop_estack = do i <- getIState
     where ptail [] = []
           ptail (x : xs) = xs
 
--- | Add a class instance function.
+-- | Add an interface instance function.
```

Comment:
Again, "interface implementation". I would try to check whether this happens other places as well.